go.mod

```diff
@@ -238,12 +236,13 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
-	sigs.k8s.io/yaml v1.3.0 // indirect
+	sigs.k8s.io/yaml v1.3.0
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.18.0-sdk-v0.46.14
+	github.com/cometbft/cometbft => /Users/callum/Developer/go/src/github.com/celestiaorg/celestia-core
```

Comment:
If you want to test this locally you will need to use `cal/rollback-hard-flag` this branch locally and redirect the replace to where your version of celestia-core is